vsts/test-release-nuget.yaml

```diff
@@ -8,10 +8,12 @@ jobs:
     displayName: Linux
     strategy:
       # Change maxParallel to 1 make builds run in serial rather than in parallel
-      maxParallel: 3
+      maxParallel: 100
```

Comment:
Do we need a limit at all? The schema says - If maxParallel is unspecified or set to 0, no limit is applied.

Reply:
good point, will make this change.